sklearn/preprocessing/_data.py

```diff
@@ -1283,6 +1293,10 @@ def fit(self, X, y=None):
 
             self.scale_ = quantiles[1] - quantiles[0]
             self.scale_ = _handle_zeros_in_scale(self.scale_, copy=False)
+            if self.unit_variance:
+                self.adjust_ = (stats.norm.ppf(q_max / 100.0) -
+                                stats.norm.ppf(q_min / 100.0))
```

Comment:
Does adjust need to be an attribute in this case?

(For me, I do not think it needs to be and `scale` is enough.

Reply:
Fair, I'll amend